src/sidebar/util/fetch-config.js

```diff
@@ -86,7 +87,7 @@ function fetchConfigLegacy(appConfig, window_ = window) {
   const hostPageConfig = hostConfig(window_);
 
   let embedderConfig;
-  const origin = hostPageConfig.requestConfigFromFrame;
+  const origin = /** @type string */ (hostPageConfig.requestConfigFromFrame);
```

Comment:
`requestConfigFromFrame` can be an object or a string, but in this particular context we know it must be a string.